flowdock-text.js

```diff
@@ -963,25 +963,29 @@ if (typeof FlowdockText === "undefined" || FlowdockText === null) {
     };
   };
 
-  FlowdockText.mentionsAllTags = ["@everyone", "@everybody", "@all", "@anyone", "@anybody"];
```

Comment:
Eventually, shouldn't this list be removed from the library altogether? I left in FlowdockText.mentionsAll and FlowdockText.mentionsTeam for compatibility reasons, but they should be eventually removed as well so that we can have business logic in other components.